tsconfig.extension.json

```diff
@@ -3,7 +3,7 @@
     "module": "commonjs",
     "target": "es6",
     "outDir": "build",
-    "lib": ["es6", "dom"],
+    "lib": ["es6", "dom", "es2019"],
```

Comment:
Is this okay for all vscode versions? What necessitated this?

Reply:
I had an issue and had to add this, don't remember what it was, maybe it resolved itself? I will delete it and see if it still works

Reply:
"It works on my machine" lmao